account/account.go

```diff
@@ -15,6 +15,14 @@ import (
 	jsonpatch "gopkg.in/evanphx/json-patch.v4"
 )
 
+type AccountPriceFloors struct {
+	Enabled           bool `mapstructure:"enabled" json:"enabled"`
+	EnforceFloorRate  int  `mapstructure:"enforce_floors_rate" json:"enforce_floors_rate"`
+	BidAdjustment     bool `mapstructure:"adjust_for_bid_adjustment" json:"adjust_for_bid_adjustment"`
+	EnforceDealFloors bool `mapstructure:"enforce_deal_floors" json:"enforce_deal_floors"`
+	UseDynamicData    bool `mapstructure:"use_dynamic_data" json:"use_dynamic_data"`
+}
```

Comment:
This object is already also in the `config` package, which is the better location IMHO.

Reply:
this shall be removed